### PR TITLE
deprecations: `aliases` are now only store-side

### PR DESCRIPTION
**Pull request by kyrofa**
More details in [this forum post](https://forum.snapcraft.io/t/improving-the-aliases-implementation/18).

### CODE REVIEW

File: deprecation-notices/dn5.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: DN2
+---
+
+**Aliases are now handled by the store, and shouldn't be placed in the snap.**
+
+_introduced in snapcraft 2.34_
```
Comment (sergiusens) [inline deprecation-notices/dn5.md]: 2.35?

Comment (kyrofa) [reply deprecation-notices/dn5.md]: Err, yeah. Nice catch.

File: deprecation-notices/dn5.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+2. Aliases could be selected piecemeal by the store and by users, which was
+   flexible but too heavy for the common case of just getting software to work.
+
+To solve these issues, aliases are no longer carried within the snap itself
+with the `aliases` property, but handled completely by the store. Users are
+free to setup aliases under their control (via `snap alias`), and automatic
+aliases are also handled as a block, being enabled or disabled together.
+
+Since the aliases are all store-side, in order to obtain them the snap
+developer much create a new topic in the [store category of the forum][1]
```
Comment (caldav) [inline deprecation-notices/dn5.md]: s/much/must

File: deprecation-notices/dn5.md (hunk 1 above)
Comment (kyrofa) [reply deprecation-notices/dn5.md]: Fixed.

File: deprecation-notices/dn5.md (hunk 2 above)
Comment (caldav) [inline deprecation-notices/dn5.md]: Suggestion for this sentence: "To create a store-side alias for a snap, the developer needs to request it and get it approved by the developer community. This process happens in the Snapcraft forum, see the [requests guidelines](https://forum.snapcraft.io/t/process-for-reviewing-aliases-auto-connections-and-track-requests/455)."

Comment (kyrofa) [reply deprecation-notices/dn5.md]: Yeah that's better. Done.

**Pull request merged**